src/test/java/org/apache/datasketches/memory/test/AllocateDirectMemoryTest.java

```diff
@@ -17,35 +17,84 @@
  * under the License.
  */
 
-package org.apache.datasketches.memory;
+package org.apache.datasketches.memory.test;
 
 import static org.testng.Assert.assertEquals;
 import static org.testng.Assert.assertFalse;
 import static org.testng.Assert.assertNotNull;
 import static org.testng.Assert.fail;
 
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+import java.nio.ByteOrder;
+
+import org.apache.datasketches.memory.MemoryRequestServer;
+import org.apache.datasketches.memory.Util;
+import org.apache.datasketches.memory.WritableDirectHandle;
+import org.apache.datasketches.memory.WritableHandle;
+import org.apache.datasketches.memory.WritableMemory;
 import org.testng.annotations.AfterClass;
 import org.testng.annotations.Test;
 
 @SuppressWarnings("javadoc")
 public class AllocateDirectMemoryTest {
-
+  
+  static final Method CHECK_VALID;
+  static final Method WRAP_DIRECT;
+  static final Method GET_CURRENT_DIRECT_MEMORY_ALLOCATIONS;
+  
+  static {
+    CHECK_VALID =
+        ReflectUtil.getMethod(ReflectUtil.BASE_STATE, "checkValid", (Class<?>[])null); //not static
+    WRAP_DIRECT =
+        ReflectUtil.getMethod(ReflectUtil.BASE_WRITABLE_MEMORY_IMPL, 
+            "wrapDirect", long.class, ByteOrder.class, MemoryRequestServer.class);  //static method
```

Comment:
The comments are a little misleading - did you mean to say that line 48 _is_ static and line 51 is _not_ a static method?

Reply:
I see - what confuses me slightly is that the class instance is null when reflecting on checkValid, which implies that it is a static method.  This is less about the commentary than making sure that I understand the distinction between static and non-static reflective access.